commit 80542e5cc9bf04267e8776b4a77262f8930f0d4a (refs/changes/14/3514/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-27T20:10:02-08:00 (1 year, 7 months ago)
    
    TRITON-193 TRITON-185 broke sapi startup: "Uncaught Error: ENOENT: no such file or directory, open '/opt/smartdc/sapi/lib/server/../../package.json'"
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/Makefile b/Makefile
index 48c0103..5a37add 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ service: all $(SMF_MANIFESTS)
 	@touch $(SVC_PKGDIR)/site/.do-not-delete-me
 	cp -r $(TOP)/server.js \
 		$(TOP)/node_modules \
+		$(TOP)/package.json \
 		$(SVC_INSTDIR)/
 	cp -r $(TOP)/lib/common \
 		$(TOP)/lib/server \
